cpanfile

```diff
@@ -28,6 +28,7 @@ requires 'Email::MIME';
 requires 'File::Basename';
 requires 'File::Path';
 requires 'File::Slurper';
+requires 'File::Which';
```

Comment:
Similar to #850 sytest doesn't seem to use this directly?

Reply:
Ah, good point. Looks like it's [Alien::Base](https://metacpan.org/pod/Alien::Base) that needs it.

But it's listed on cpan... so why isn't it installed? :thinking:

Reply:
It's not immediately obvious to the team, so I'll give this another run in a fresh VM, get some logs and see if we can track it down.

Reply:
Running this again in a fresh VM and installing the native dependencies mentioned in this PR, I received no dependency issues and was able to do a full successful sytest run.

I'll update the PR with only those.